hatchways/team-gingerbread
# FS: Reviews

### What this PR does (required):
- addresses #42 
- creates reviews container on profile details page, as well as review form to leave a review


### Screenshots / Videos (front-end only):
![image](https://user-images.githubusercontent.com/78042694/148841502-8d824af0-3e9a-4acf-b0c9-1a06c2e75c9d.png)


### Any information needed to test this feature (required):
- run server and app
- navigate to profile details page 
   - http://localhost:3000/profile-details/<sitter's user id>
- submit a review via the form on the right hand side
- after page reload, your review should appear under the sitter's profile details on the left-hand side
### Any issues with the current functionality (optional):
- currently does not display reviewer's name (need to add in fetchProfile call with clientId)
- so far, anyone can leave a review whether they have booked the sitter or not (can change this once everything is integrated)
- need to add in functionality on profile details page to change user's rating each time a new review is submitted for them
